cypress/integration/features/login_management/passwordUpdate.feature

```diff
@@ -0,0 +1,48 @@
+@login @stable_test
+Feature: Password Update - Verify login functionality via password change
+
+    As a user, I want to see how app behaves when I'll change password.
+
+    Scenario: User change password, login and change again to old passwords.
+        Given I am on the dBildungscloud login page
+        Then I will see Email field is visible and empty
+        Then I will enter email
+        Then I will see Password field is visible and empty
+        Then I will enter password
+        Then I will click submit button
+        When I go to my account settings
+        Then I will see current password field is visible and empty
+        Then I will see new and repeat password field is visible and empty
+        When I enter current password
+        Then I enter new password in both fields
+        Then I will click on submit button
```

Comment:
this line can be more unique to save in this scope -> 'Then I click on the save account settings button'